src/commands/commerce/extension/register.ts

```diff
@@ -148,6 +181,12 @@ export class RegisterExtension extends SfdxCommand {
                 ExtensionPointName: element['ExtensionPointName'] as string,
                 ExternalServiceProviderType: element['ExternalServiceProviderType'] as string,
             };
+
+            if (parseInt(this.flags['apiversion'], 10) >= 59) {
+                finalTable['Description'] = element['Description'] as string;
+                finalTable['IconURI'] = element['IconUri'] as string;
+                finalTable['IsApplication'] = element['IsApplication'] as string;
```

Comment:
Isn't this field boolean?

Reply:
Like I mentioned in the earlier comment, this part is just for printing so we are casting everything to string.

Reply:
Is there any better way to do this instead of hardcoding the version number? ls there a feature toggle?